src/pages/sign-transaction/components/fee-form.tsx

```diff
@@ -30,32 +35,40 @@ export function FeeForm(): JSX.Element | null {
   const [, setFeeEstimations] = useFeeEstimationsState();
 
   useEffect(() => {
-    if (feeEstimationsResp && feeEstimationsResp.estimations) {
-      setFeeEstimations(feeEstimationsResp.estimations);
-      setFieldValue(
-        'fee',
-        stacksValue({
-          fixedDecimals: true,
-          value: feeEstimationsResp.estimations[Estimations.Middle].fee,
-          withTicker: false,
-        })
-      );
+    if (feeEstimationsResp) {
+      if (
+        (isError || !!feeEstimationsResp?.error || !feeEstimationsResp.estimations.length) &&
+        estimatedSignedTxByteLength
+      ) {
+        const fee = calculateFeeFromFeeRate(estimatedSignedTxByteLength, DEFAULT_FEE_RATE);
+        setFeeEstimations([
+          { fee: fee.multipliedBy(0.9).toNumber(), fee_rate: 0 },
+          { fee: fee.toNumber(), fee_rate: 0 },
+          { fee: fee.multipliedBy(1.1).toNumber(), fee_rate: 0 },
+        ]);
```

Comment:
Yeah, I'll go back thru and simplify this so it is a bit more dry. I don't see anything wrong with the fee rate being `0` since we aren't using it atm. We can always change it later?

Reply:
I think 0 is good. If we started using it and it tried to broadcast a tx with 0 fee, it'll fail. So there's no consequence to be concerned about.